src/util/datablock/datablock.c

```diff
@@ -48,19 +48,16 @@ static void _DataBlock_AddBlocks(DataBlock *dataBlock, uint blockCount) {
 	dataBlock->itemCap = dataBlock->blockCount * DATABLOCK_BLOCK_CAP;
 }
 
-static inline void _DataBlock_MarkItemAsDeleted(const DataBlock *dataBlock, unsigned char *item) {
-	memset(item, DELETED_MARKER, dataBlock->itemSize);
+static inline void _DataBlock_MarkItemAsDeleted(DataBlockItemHeader *item_header) {
+	MARK_HEADER_AS_DELETED(item_header);
 }
 
-static inline void _DataBlock_MarkItemAsUndelete(const DataBlock *dataBlock, unsigned char *item) {
-	item[0] = !DELETED_MARKER;
+static inline void _DataBlock_MarkItemAsUndelete(DataBlockItemHeader *item_header) {
```

Comment:
Do you think we need this "proxy" function now that we have a macro?

Reply:
removed